src/NuGet.Clients/NuGet.CommandLine/Common/CommandLineRepositoryFactory.cs

```diff
@@ -31,7 +31,17 @@ public override IPackageRepository CreateRepository(string packageSource)
                             System.ConsoleColor.Green,
                             "{0} {1}", args.Request.Method, args.Request.RequestUri);
                     }
-                    string userAgent = HttpUtility.CreateUserAgentString(CommandLineConstants.UserAgent);
+
```

Comment:
Addressed in https://github.com/NuGet/NuGet.Client/commit/906824e17107215a009984693f979465c411132d